op-deployer/pkg/deployer/artifacts/download.go

```diff
@@ -51,7 +51,14 @@ func Download(ctx context.Context, loc *Locator, progressor ioutil.Progressor, t
 			return nil, fmt.Errorf("failed to download artifacts: %w", err)
 		}
 	case "file":
-		artifactsFS = os.DirFS(u.Path)
+		// Check the path has forge-artifacts directory
+		forgeArtifactsDir := path.Join(u.Path, "forge-artifacts")
+		if _, err := os.Stat(forgeArtifactsDir); err != nil {
+			// Accept this for now but in the future we should error
```

Comment:
Can we add a TODO(#XXXXX) tracker here?

Reply:
Added #18346 to track this issue, it requires some refactoring.